backend/api/apis/configuration/v1alpha1/configuration.go

```diff
@@ -0,0 +1,18 @@
+package v1alpha1
+
+import (
+	"github.com/google/uuid"
+)
+
+type Configuration struct {
```

Comment:
can you comment indicating what mean every member of struct

Reply:
Sure! Goinggg